src/index.js

```diff
@@ -75,10 +75,9 @@ const API: TAsyncStorage = {
         }
 
         try {
-          const parsedValue = JSON.parse(rawValue);
```

Comment:
From RN docs of [multiMerge](https://facebook.github.io/react-native/docs/asyncstorage.html#multimerge): `This assumes that the values are stringified JSON.`. And if we check typing (for both `setItem` and `multiSet`) value it's always string, so if we want to pass and object, the assumption is correct (we would previously have already used `JSON.stringify()`).